tutorials/xr/a_better_xr_start_script.rst

```diff
@@ -192,8 +192,8 @@ it is nicer to exit on failure than to hang the system.
     ...
 
 
-On session begun
-----------------
+_on_openxr_session_begun
```

Comment:
```suggestion
.. _on_openxr_session_begun:

On session begun
```
For all these, but only add the ones that are actually referenced

Reply:
Unless you want it to link to somewhere else, this wouldn't work here I think, but that would be:
```
 :ref:`On session begun <TARGET>`
```
But there's nothing to link to in the code like that, and it's just there so I don't think that would make sense

Tbh I don't know what you're trying to achieve

Reply:
These sections all begin with "This signal" without mentioning the name of the specific signal before or after. Specific  name can only be recognized by looking at the code more closely.

I'm not sure if first time XR developers with Godot will have the same problem when they see this paragraph? So it may no need to linking to other places, the modification is just to replace section title to mention the signals.

Reply:
But the name of the signal is the title, it's `session_begun`, not `_on_openxr_session_begun`

I don't think this change makes sense, the only think I'd say would make sense is to link to the signal, which would be:
`class_OpenXRInterface_signal_session_begun`

Reply:
I agree that we shouldn't get too hung up on details like that, and I'm not particularly deep into the source code, apologise for taking up some of your time. 🙏